specs/interop/shared-lockbox-upgrade.md

```diff
@@ -0,0 +1,128 @@
+# Shared Lockbox - Upgrade and migration process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Add the chain to the op-governed dependency set](#add-the-chain-to-the-op-governed-dependency-set)
+  - [Migrate ETH liquidity from `OptimismPortal` to `SharedLockbox`](#migrate-eth-liquidity-from-optimismportal-to-sharedlockbox)
+    - [`LiquidityMigrator`](#liquiditymigrator)
+  - [`OptimismPortal` code upgrade](#optimismportal-code-upgrade)
+- [Batch transaction process](#batch-transaction-process)
+  - [Diagram](#diagram)
+- [Future Considerations / Additional Notes](#future-considerations--additional-notes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+Based on the assumption that a chain joining the dependency set is an irreversible process,
+the on-chain chains list is simplified by assuming that joining the Shared Lockbox is
```

Comment:
I lack some context here, what/where is the on-chain chains list? 
It seems like it's becoming equivalent to the dependency set, was it previously different?